src/components/NewClusterModal.js

```diff
@@ -216,209 +375,77 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
-                ]),
-                div({ style: { gridColumnStart: 2, gridColumnEnd: 'span 2', alignSelf: 'start' } }, [
-                  h(Link, { href: imageInstructions, ...Utils.newTabLinkProps }, ['Custom notebook environments']),
-                  span({ style: { fontWeight: 'bold' } }, [' must ']),
-                  ' be based off one of the ',
-                  h(Link, { href: terraBaseImages, ...Utils.newTabLinkProps }, ['Terra base images.'])
-                ])
-              ]) :
-              h(Fragment, [
-                h(IdContainer, [
-                  id => h(Fragment, [
-                    label({ htmlFor: id, style: styles.label }, 'Environment'),
-                    div({ style: { gridColumnEnd: 'span 2', height: '45px' } }, [
-                      makeEnvSelect(id)
-                    ])
-                  ])
-                ]),
-                div({ style: { gridColumnStart: 2, alignSelf: 'start' } }, [
-                  h(Link, { onClick: () => this.setState({ viewMode: 'Packages' }) }, ['What’s installed on this environment?'])
-                ]),
-                makeImageInfo()
-              ])
-          ]),
-          div({
-            style: {
-              padding: '1rem', marginTop: '1rem',
-              backgroundColor: colors.dark(0.15),
-              border: `2px solid ${colors.dark(0.3)}`, borderRadius: '9px'
-            }
-          }, [
-            div({ style: { fontSize: '0.875rem', fontWeight: 600, marginBottom: '0.5rem' } }, ['COMPUTE POWER']),
-            div({ style: { marginBottom: '1rem' } }, ['Select from one of the compute runtime profiles or define your own']),
-            div({ style: { display: 'grid', gridTemplateColumns: '1fr 1fr 1fr 1.2fr 1fr 5.5rem', gridGap: '1rem', alignItems: 'center' } }, [
-              h(IdContainer, [
-                id => h(Fragment, [
-                  label({ htmlFor: id, style: styles.label }, 'Profile'),
-                  div({ style: { gridColumnEnd: 'span 5' } }, [
-                    h(Select, {
-                      id,
-                      value: profile,
-                      onChange: ({ value }) => {
-                        this.setState({
-                          profile: value,
-                          ...(value === 'custom' ? {} : normalizeMachineConfig(_.find({ name: value }, profiles).machineConfig))
-                        })
-                      },
-                      isSearchable: false,
-                      isClearable: false,
-                      options: [
-                        ..._.map(({ name, label }) => ({ value: name, label: `${label} computer power` }), profiles),
-                        { value: 'custom', label: 'Custom' }
-                      ]
-                    })
+                    div({ style: { marginBottom: '0.5rem' } }, [label({ htmlFor: id, style: styles.label }, 'ENVIRONMENT')]),
+                    div({ style: { height: '45px' } }, [makeGroupedEnvSelect(id)])
```

Comment:
The validation error ends up a little cramped with this at 45px. We should bump it up to at least 52px or so.